packages/block-library/src/post-date/block.json

```diff
@@ -2,7 +2,7 @@
 	"$schema": "https://schemas.wp.org/trunk/block.json",
 	"apiVersion": 3,
 	"name": "core/post-date",
-	"title": "Post Date",
+	"title": "Date",
 	"category": "theme",
 	"description": "Add the date of this post.",
```

Comment:
Maybe for a follow-up, but what about the word "post"  in the descriptions?

Reply:
Good point. Would making that less explicit work?

> Display the publish date for an entry such as a post or page.

Reply:
> Display the publish date for an entry such as a post or page.

Done.